build_tools/cmake/test_riscv.sh

```diff
@@ -38,6 +38,8 @@ declare -a label_exclude_args=(
   "^vulkan_uses_vk_khr_shader_float16_int8$"
   "^requires-filesystem$"
   "^requires-dtz$"
+  # Tests labelled noriscv can cause timeouts when run in emulator.
+  "^noriscv$"
```

Comment:
nit: I'd leave out "can cause timeouts" from the comment in the script, as the label could be added for other reasons